components/app-core/backend/repository/tokens/pgsql_tokens.go

```diff
@@ -278,9 +284,18 @@ func (p *PgsqlTokenRepository) SaveCNSIToken(cnsiGUID string, userGUID string, t
 	return nil
 }
 
-// FindCNSIToken - retrieve a CNSI (UAA) token from the datastore
 func (p *PgsqlTokenRepository) FindCNSIToken(cnsiGUID string, userGUID string, encryptionKey []byte) (interfaces.TokenRecord, error) {
 	log.Println("FindCNSIToken")
+	return p.findCNSIToken(cnsiGUID, userGUID, encryptionKey, false)
+}
+
+func (p *PgsqlTokenRepository) FindCNSITokenIncludeDisconnected(cnsiGUID string, userGUID string, encryptionKey []byte) (interfaces.TokenRecord, error) {
+	log.Println("FindCNSITokenIncludeDisconnected")
```

Comment:
Can we change log.Println to log.Debug, otherwise we're going to see a lot of log junk.